tests/langtest_c.rs

```diff
@@ -119,6 +119,7 @@ fn mk_compiler(exe: &Path, src: &Path, opt: &str, extra_objs: &[PathBuf]) -> Com
         "-Wl,--mllvm=--disable-early-taildup", // Interferes with the BlockDisambiguate pass.
         "-Wl,--mllvm=--disable-tail-duplicate", // ^^^
         "-Wl,--mllvm=--yk-disable-tail-call-codegen", // Interferes with the JIT's inlining stack.
+        "-Wl,--mllvm=--yk-no-fallthrough",     // Distorts the block mapping.
```

Comment:
This flag doesn't distort the block mapping? Not using it distorts the mapping. So it un-distorts the block mapping.

Reply:
fixed in [d142b30](https://github.com/ykjit/yk/pull/533/commits/d142b30250c2aa94dc7b203adc200c20630a1557)